### ISSUE TITLE
Memory Exception -- restart with less ensemle_nbest..

### ISSUE THREAD

**Issue by StevenLOL** [automl/auto-sklearn#513]
No matter what ml_memory_limit is, I will get the memory exceptions: 
```
auto=AutoSklearnClassifier(include_estimators=['lda','k_nearest_neighbors','passive_aggressive','gradient_boosting','extra_trees','liblinear_svc'], 
                           ml_memory_limit=500000,    # yes around 500GB
                              resampling_strategy='cv',
                            resampling_strategy_arguments={'folds': 5},
                           #time_left_for_this_task=120,per_run_time_limit=60,
                           
                          )
```

```
[WARNING] [2018-07-20 11:26:15,141:EnsembleBuilder(1):bc7ea84b06c06db9da18252766d56c64] Memory Exception -- restart with less ensemle_nbest: 25
[WARNING] [2018-07-20 11:26:16,498:EnsembleBuilder(1):bc7ea84b06c06db9da18252766d56c64] Memory Exception -- restart with less ensemle_nbest: 12
[WARNING] [2018-07-20 11:26:17,787:EnsembleBuilder(1):bc7ea84b06c06db9da18252766d56c64] Memory Exception -- restart with less ensemle_nbest: 6
[WARNING] [2018-07-20 11:26:19,135:EnsembleBuilder(1):bc7ea84b06c06db9da18252766d56c64] Memory Exception -- restart with less ensemle_nbest: 3
[WARNING] [2018-07-20 11:26:20,575:EnsembleBuilder(1):bc7ea84b06c06db9da18252766d56c64] Memory Exception -- restart with less ensemle_nbest: 1
```

**Comment by mfeurer** [comment]
The memory limit argument only applies to running the actual machine learning algorithm, not the ensemble. You need to change the limit in the file `ensemble_builder` manually for now.

Just out of curiosity: how big is your dataset?

**Comment by StevenLOL** [comment]
@mfeurer ,

train.shape= (125973,8)
dev.shape= (22544, 8)

**Comment by mfeurer** [comment]
Will be fixed in the upcoming release via ab45a25a.
